src/commands/cloneSite.js

```diff
@@ -0,0 +1,54 @@
+const axios = require('axios')
+const chalk = require('chalk')
+const config = require('../config/config')
+const path = require('path')
+
+async function loadWebsiteScraper () {
+  // Dynamically import the ES module `website-scraper`
+  return await import('website-scraper')
+}
+
+async function cloneSite (siteId, outputDir) {
+  try {
+    console.log(chalk.blue(`Cloning site with ID: ${siteId}`))
+
+    // Trigger server-side cloning
+    await axios.post(`${config.dpApiUrl}/sites/${encodeURIComponent(siteId)}/clone`, null, {
+      headers: {
+        Authorization: `Bearer ${config.authToken}`,
+        'Content-Type': 'application/json'
+      },
+      validateStatus: (status) => status >= 200 && status < 300
+    })
+
+    console.log(chalk.green('Server-side cloning initiated successfully!'))
+
+    // Proceed to download the site content locally
+    const outputDirectory = outputDir || path.join(process.cwd(), siteId)
+
+    let siteUrl = siteId
+    if (!siteUrl.startsWith('http://') && !siteUrl.startsWith('https://')) {
+      siteUrl = `https://${siteUrl}`
+    }
+
+    console.log(chalk.blue(`Downloading site content from ${siteUrl} to ${outputDirectory}...`))
+
+    // Dynamically load `website-scraper`
+    const { default: scrape } = await loadWebsiteScraper()
```

Comment:
We shouldn't be doing a scrape/download, that's what the clone HTTP API does already. IT's enough to trigger the clone and just output the site URLs

Reply:
https://github.com/hyphacoop/distributed-press-cli/pull/1/commits/365efb3ee7f081718241b7ba829a37eef2908208

`dp-cli clone ranger.mauve.moe`
```
Cloning site with ID: ranger.mauve.moe
Server-side cloning initiated successfully!
Cloned Site Info:
{
  "id": "ranger.mauve.moe",
  "domain": "ranger.mauve.moe",
  "protocols": {
    "http": true,
    "hyper": true,
    "ipfs": true
  },
  "links": {
    "http": {
      "enabled": true,
      "link": "https://ranger.mauve.moe"
    },
    "hyper": {
      "gateway": "https://ranger-mauve-moe.hyper.hypha.coop/",
      "raw": "hyper://xxsog44z58458te7gkxgrggmxwastm94om6d9n1hjsjtsi7rt9wo/",
      "dnslink": "/hyper/xxsog44z58458te7gkxgrggmxwastm94om6d9n1hjsjtsi7rt9wo/",
      "enabled": true,
      "link": "hyper://ranger.mauve.moe"
    },
    "ipfs": {
      "gateway": "https://ranger-mauve-moe.ipns.ipfs.hypha.coop",
      "cid": "bafybeibwjxk563q3lw226qdlptwzvjk7skyfsbblqa4jc4ulhuftcevqsi",
      "pubKey": "ipns://k51qzi5uqu5dkyebpkw5k4kkecddn0bnd9zch18g9bpb4htoulp439uyb4gugc/",
      "dnslink": "/ipns/k51qzi5uqu5dkyebpkw5k4kkecddn0bnd9zch18g9bpb4htoulp439uyb4gugc/",
      "enabled": true,
      "link": "ipns://ranger.mauve.moe/"
    }
  },
  "public": true
}
```